wiki/_Sidebar.md

```diff
@@ -30,6 +30,7 @@
 * [Creating new ships](CreatingShips)
 * [Creating person ships](CreatingPersons)
 * [Creating outfits and weapons](CreatingOutfits)
+* [Creating shops](Creating-Shops)
```

Comment:
Is it possible to handle this in a separate PR? (Just so that it doesn't need to wait for the discussions on this PR to finish?)